docs/coauthors.me/src/components/CoauthorsGenerator.tsx

```diff
@@ -97,25 +97,51 @@ export const CoauthorsGenerator = () => {
                   <button
                     type="button"
                     onClick={() => setAuthors((prev) => prev.filter((authors) => authors.user !== user))}
-                    className="h-10 w-[50px] select-none rounded"
+                    className="h-10 w-[50px] select-none rounded bg-black"
                   >
                     🗑️
                   </button>
                 </div>
               </div>
             </QueryErrorBoundary>
           ))}
+          {authors.length === 0 ? null : <CopyButton authors={authors} />}
+          {authors.length > 0 ? <p className="text-center opacity-60">⌨️ Copy & Paste on commit message ⌨️</p> : null}
```

Comment:
```suggestion
      {authors.length > 0 ? (
        <>
          <CopyButton authors={authors} />
          <p className="text-center opacity-60">
            ⌨️ Copy & Paste on commit message ⌨️
          </p>
        </>
      ) : null}
```
How about simplifying it based on conditions ?

Reply:
👍